keras/src/backend/openvino/numpy.py

```diff
@@ -1117,8 +1117,52 @@ def pad(x, pad_width, mode="constant", constant_values=None):
     )
 
 
+"""
+Helper Function to convert the string dtype to ov type
+"""
+
+
+def string_to_ov_type(dtype_str):
+    from openvino.runtime import Type
+
+    mapping = {
+        "bool": Type.boolean,
+        "int8": Type.i8,
+        "int16": Type.i16,
+        "int32": Type.i32,
+        "int64": Type.i64,
+        "uint8": Type.u8,
+        "uint16": Type.u16,
+        "uint32": Type.u32,
+        "uint64": Type.u64,
+        "float16": Type.f16,
+        "float32": Type.f32,
+        "float64": Type.f64,
+    }
+    return mapping[dtype_str]
```

Comment:
please re-use this dictionary: https://github.com/keras-team/keras/blob/master/keras/src/backend/openvino/core.py#L25
Yours is not needed